lib/i18n/hygiene/wrapper.rb

```diff
@@ -8,7 +8,8 @@ module Hygiene
     # queryable.
     class Wrapper
 
-      def initialize(keys_to_skip:)
+      def initialize(keys_to_skip: [], locales: ::I18n.available_locales)
```

Comment:
It seems to be set automatically when you load/store translations, for example in the spec I wrote:
```
        [:fr, :es].each do |locale|
          ::I18n.backend.send(:store_translations, locale, abuse_report: { button: { submit: "x" }})
        end
```

If you omit the `locales: [:en, :fr]` in the constructor, it includes them all via `I18n.available_locales`.

I might expand that test to make that more obvious.